common-pg/src/main/scala/com/socrata/pg/analyzer2/ExprSqlizer.scala

```diff
@@ -18,8 +18,7 @@ class ExprSqlizer[MT <: MetaTypes](
       // literal id/version will have a null provenance, but one of
       // those ending up in a set of values being given to an ORDER BY
       // is _such_ an edge condition that I honestly don't care.
-      val possibleProv = dynamicContext.provTracker(e.expr)
-      if(possibleProv.size < 2) {
+      if(!dynamicContext.provTracker(e.expr).isPlural) {
```

Comment:
Oh actually the comment here is out of date; the new thing handles that case regularly.